spec.bs

```diff
@@ -4208,13 +4206,16 @@ To <dfn>get the value to report from an averager</dfn> given an [=averager=] |av
 </div>
 
 ### Metrics structures ### {#private-aggregation-metrics-structures}
-An <dfn>execution metrics</dfn> is a [=struct=] with the following [=struct/items=]:
+An <dfn>execution metrics</dfn> is a [=struct=] with the following [=struct/items=], representing
+metrics collected from a single execution of a worklet function:
 <dl dfn-for="execution metrics">
   : <dfn>code fetch time averager</dfn>
   :: An [=averager=].
 </dl>
 
-A <dfn>per participant metrics</dfn> is a [=struct=] with the following [=struct/items=]:
+A <dfn>per participant metrics</dfn> is a [=struct=] with the following [=struct/items=],
+representing metrics aggregated over everything a particular participant does in an auction,

```

Comment:
maybe just me, but it reads like it aggregated a metric from both generateBid() and scoreAd() together (e.g., code fetch time). I think it maybe more important to make it clear that this is reported once per participant, not per ig?

Reply:
would saying (e.g. a bidder or a seller) after the 'a particular participant' help any?

I don't want to talk about "per ig" since it's not really accurate from sellers PoV in the first place.

Reply:
Done..